examples/activities/resample.js

```diff
@@ -62,19 +70,31 @@ module.exports = function resample(localInputPath, outputUri, options, callback)
         return done.apply(null, arguments);
       }
 
+      // try {
```

Comment:
My tiling algorithm doesn't really produce cells that don't intersect with images, so this step wasn't necessary. Also, when I removed it, I cut out a lot of time...this is pretty slow.